### PR TITLE
Fix floating promises warings 

### PR DESCRIPTION
**Pull request by unsortedhashsets**
Init issues Relate to #461:

**Fixed with added async/await**
~~/home/runner/work/camel-dap-client-vscode/camel-dap-client-vscode/src/extension.ts
Warning: 40:2 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises
Warning: 72:8 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises
Warning: 92:2 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises
Warning: 103:2 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises~~

**Fixed with catch**
~~/home/runner/work/camel-dap-client-vscode/camel-dap-client-vscode/src/test/runTest.ts
Warning: 48:1 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises~~

**Fixed with added async/await**
~~/home/runner/work/camel-dap-client-vscode/camel-dap-client-vscode/src/test/suite/completion.task.test.ts
Warning: 38:4 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises~~

**Fixed with catch**
~~/home/runner/work/camel-dap-client-vscode/camel-dap-client-vscode/src/ui-test/uitest_runner.ts
Warning: 45:1 warning Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator @typescript-eslint/no-floating-promises~~

heavy_multiplication_x 7 problems (0 errors, 7 warnings)

### CODE REVIEW

File: src/test/suite/completion.task.test.ts [hunk 1]
```diff
@@ -3,14 +3,14 @@ import * as vscode from 'vscode';
 import { getDocUri } from './util';
 
 suite('Should do completion in tasks.json', () => {
-	
-	
+
```
Comment (djelinek) [inline src/test/suite/completion.task.test.ts:6]: why two empty lines? one should be enough 🙂

Comment (unsortedhashsets) [reply src/test/suite/completion.task.test.ts:6]: Lol, there were two lines prettier just delete tab xD, will fix it too

Comment (unsortedhashsets) [reply src/test/suite/completion.task.test.ts:6]: BTW, I would like to delete both lines, I think there is no need in one also

Comment (unsortedhashsets) [reply src/test/suite/completion.task.test.ts:6]: BTW, I would like to delete both lines, I think there is no need in one also

Comment (djelinek) [reply src/test/suite/completion.task.test.ts]: it depends, but ok

**Review by djelinek — APPROVED** [review]

**Pull request merged**